presto-spi/src/main/java/com/facebook/presto/spi/Node.java

```diff
@@ -32,4 +32,6 @@
     String getVersion();
 
     boolean isCoordinator();
+
+    boolean isResourceManager();
```

Comment:
a `NodeType` enum might be worthwhile to introduce at some point.

Reply:
What do you think of the suggestion here?

https://github.com/prestodb/presto/pull/15479#discussion_r549898972